manifests/init.pp

```diff
@@ -132,58 +132,51 @@
   String                                  $service_name              = $monit::params::service_name,
   Optional[Integer[1]]                    $start_delay               = $monit::params::start_delay,
 ) inherits monit::params {
-  # <stringified variable handling>
-  if is_string($httpd) == true {
+  if $httpd =~ String {
     $httpd_bool = str2bool($httpd)
   } else {
     $httpd_bool = $httpd
   }
```

Comment:
Maybe it's better to change to
```puppet
  $httpd_bool = Boolean($httpd)
```

Reply:
I decided to do this via codebase hardening of the parameter types